htdocs/libraries/icms/messaging/MentionService.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+declare(strict_types=1);
+
+namespace icms\messaging;
+
+use icms_messaging_Handler;
+use PDO;
+use Throwable;
+
+/**
+ * MentionService
+ * - Extract mentioned users from rich text content
+ * - Notify mentioned users via ImpressCMS messaging (PM)
+ *
+ * Conventions
+ * - Parses plain text @username tokens
+ * - Parses legacy HTML links to user profiles containing uid query param
+ * - Resolves only active users (level > 0)
+ */
+final class MentionService
```

Comment:
This is not an issue, but I prefer the following naming rules for services:

1. If a class has only static methods, its name should end with `Helper`.  
2. If a class can be given a meaningful name that clearly exposes its domain without using `Service`, use that name (f.e. `FileResolver`, `SlackNotifier`...).  
3. If it cannot, then its name should end with `Service`.

So, at the end there would be not too much services that ends with `Service`

Reply:
Good points! I'll take this rule over in the documentation somewhere so that we can remain consistent.